content/projects/oop/bank-accounts/part-2/_index.md

```diff
@@ -62,19 +62,20 @@ Update your directory structure to:
 ```
 ...
 
-├── src
-|   ├── main
-|   |   ├── java
-|   |   |    ├── banking
-|   |   |    |    └── BankAccount.java
-|   |   |    |    └── Bank.java <-- NEW
-
+└── app
```

Comment:
Should we not matc what we've put on other pages, like indicating the rest of the gradle config files and folders like we did here:

![image](https://github.com/Umuzi-org/ACN-syllabus/assets/57295227/bb37a5f0-a48c-40f4-8d3d-e1dc0239cf7c)